### DISCUSSION TITLE
building-a-blog-from-scratch/

### DISCUSSION THREAD

**Discussion by giscus** [Philo-Li/philo-li.github.io#2] [General]
# building-a-blog-from-scratch/

你是不是早就厌倦了那些博客网站毫无美感的界面，是不是受够了无休无止的网站推送，是不是早就想建立一个属于自己的博客，却止步于复杂的教程以及一系列令人头疼的代码？那么恭喜你，这篇文章就是想以最浅显易懂的方式手把手教你搭建属于自己的博客，你只需要一点耐心，一步步跟着做。

http://philoli.com/building-a-blog-from-scratch/

**Comment by Philo-Li** [comment]
test

**Comment by Learner110110** [comment]
捉虫：域名购买部分的链接点击后 404

**Reply by Philo-Li** [reply]
感谢！已修正！

**Comment by angping** [comment]
建议使用新版主题:https://github.com/theme-next/hexo-theme-next themes/next，博主这个仓库的版本已经停止更新了。
